code/client/clrcore/EventHandlerDictionary.cs

```diff
@@ -48,6 +59,14 @@ internal async Task Invoke(string eventName, string sourceString, object[] argum
             
             if (TryGetValue(lookupKey, out entry))
             {
+#if IS_FXSERVER
+
+                if (m_serverInternalNetEvents.Contains(eventName) && !sourceString.StartsWith("internal-net"))
+                {
+                    Debug.WriteLine($"Event {eventName} was not safe for net.");

```

Comment:
Where's the rejection?

I consider it _somewhat_ unlikely that any script will be calling these and expecting them to work, and even a log line can be seen as a compatibility break.

Reply:
Whoops, forgot to add the `return` statement 🤦‍♂️
Will include that with your other requested changes 😅